ext/gtest/CMakeLists.txt.in

```diff
@@ -10,6 +10,7 @@ ExternalProject_Add(googletest
     BINARY_DIR "${CMAKE_BINARY_DIR}/googletest-build"
     CMAKE_ARGS -DCMAKE_ARCHIVE_OUTPUT_DIRECTORY_DEBUG:PATH=DebugLibs
                -DCMAKE_ARCHIVE_OUTPUT_DIRECTORY_RELEASE:PATH=ReleaseLibs
+    UPDATE_DISCONNECTED 1
```

Comment:
Where are we using the variable?

Reply:
When cmake, this avoid to git fetch and check whether the local revision of gtest is the latest.
I think it is not necessary, and too time-consuming.
e.g `make cpptest` will be fast if we config this.